linera-execution/src/lib.rs

```diff
@@ -129,6 +129,8 @@ pub enum ExecutionError {
     OwnerIsNone,
     #[error("Application is not authorized to perform system operations on this chain: {0:}")]
     UnauthorizedApplication(UserApplicationId),
+    #[error("Failed to make network reqwest")]
```

Comment:
```suggestion
    #[error("Failed to make network request")]
```
Or "request with reqwest"?

Reply:
No you're right, callers don't care about the implementation details of how a request is made.